ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/ViewScanConfigDetail.tsx

```diff
@@ -196,7 +200,7 @@ function ViewScanConfigDetail({
                     </Tabs>
                 </PageSection>
             )}
-            {selectedTab === 'CONFIGURATION_DETAILS' && (
+            {activeScanConfigTab === 'CONFIGURATION_DETAILS' && (
```

Comment:
Just as an alternative pattern that we as a team might want to evaluate, how robust to design changes is it to lift up `PageSection` element and then conditionally render `<TabContentId id="whatever">` which as far as I could find has no effect on layout.

Reply:
I mean we could do that. The only downside is another instance of conditional logic to determine the `id` value. I'm not sure if it would be cleaner than this approach though.